### ISSUE TITLE
Issue when referencing full URL

### ISSUE THREAD

**Issue by andrewicarlson** [kvendrik/responsive-images.js#8]
I believe I've found a (potential) bug when referencing a full URL such as http://example.com/images/large.jpg. Seemingly whenever a full URL including a colon is used in the 'data-src', it stops the dynamic generation of the 'src' attribute before the colon.

Eg: If I have data-src="<320:http://example.com/images/small.jpg" it will generate src="http" rather than src="http://example.com/images/small.jpg"

I believe this to just be an issue with the way it parses colons, but it's late and I won't have time to look at this until tomorrow.

**Comment by andrewicarlson** [comment]
So after looking through the code a bit, I'm fairly confident that this is stemming from line 77 in the "non-minified" version (responsive-img.js). 

`var query = queries_array[j].split(':');`

In order to maintain backward compatibility and not require an instruction rewrite, I would think that a regex needs to be used in the split to only match `:` preceded by a digit (for the breakpoint).

I don't have time to test this now but I will tonight.

**Comment by kvendrik** [comment]
Hi @aicarlson and @adambom,

Sorry for my late response. I didn't find time to fix this until now.

Since a regex split doesn't seem to work in IE6 I created a simple solution that will first replace ':' with '||' and then splits the string at '||'. 

The problem was there in the first place because the split methods splits the string at all occurences of ':' it finds. The replace method on the other hand only replaces the first occurrence of the character it finds.

So in short:

```
var query = queries_array[j].split(':');
```

is now replaced by

```
var query = queries_array[j].replace(':','||').split('||');
```

This solution is not perfect and I will look into a better one soon. Let me what you think and if you have ideas! :)
